guides/common/modules/proc_configuring-satellite-with-an-alternate-cname.adoc

```diff
@@ -0,0 +1,16 @@
+[id='configuring-satellite-with-an-alternate-cname_{context}']
+= Configuring {Project} with an Alternate CNAME
+Use this procedure to configure {Project} with an alternate CNAME. Note that the procedures for users of a default {Project} certificate and custom certificate differ.
```

Comment:
Also note that this only works on Katello, not vanilla Foreman.

Reply:
Noted this at the beginning of the procedure